doc/manual/book.toml

```diff
@@ -1,2 +1,6 @@
 [output.html]
 additional-css = ["custom.css"]
+additional-js = ["redirects.js"]
+
+[preprocessor.anchors]
+command = "python3 doc/manual/anchors.py"
```

Comment:
This introduces a dependency on Python, which is currently not listed in `flake.nix` or checked for in the configure script. Python *is* available in our dev shell but I'm not sure where it comes from. The Nix derivation in Nixpkgs doesn't seem to depend on Python. It would be a bit sad to require Python just for this...

Reply:
It appears to come from mercurial. I can add it to `flake.nix` explicitly.

Alternately, I can rewrite the script in `jq` or some other language (it just needs JSON and regex replacement). But Python will still be in build closure through rustc « mdbook.

Reply:
If it can be done with `jq` without too much effort, that would be great.